tool/tsh/common/proxy_test.go

```diff
@@ -249,6 +251,60 @@ func TestSSHLoadAllCAs(t *testing.T) {
 	}
 }
 
+// TestWithRsync tests that Teleport works with rsync.
+func TestWithRsync(t *testing.T) {
```

Comment:
This test should be moved to integration tests IMO.

Reply:
Since this is strictly testing `tsh`, I think it makes sense to leave it where it is. Especially now that part of the test relies on re-executing the test binary as `tsh`.